### PR TITLE
Allow localised dividers in radio button groups

### PR DESCRIPTION
**Pull request by HughePaul**

### CODE REVIEW

File: lib/globals.js [hunk 1]
```diff
@@ -108,6 +108,18 @@ let globals = {
...
+
+            if (item.divider) {
+                if (typeof item.divider !== 'string') {
+                    let key = item.key || [
+                        contentKey + '.divider.label',
+                        'fields.default.divider.label'
+                    ];
+                    item.divider = translate(key);
+                }
+                return item;
```
Comment (merlinc) [inline lib/globals.js:120]: Just to be clear, are we returning the `item` here because if its a divider, there's nothing else we can do with this data?

Comment (HughePaul) [reply lib/globals.js:120]: The govuk component just renders a div with the text, it doesn't even support classes or IDs

https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/components/radios/template.njk#L56-L58

Comment (MMcC-GDS) [reply lib/globals.js:120]: How has this been tested?
I’m struggling to get it working on the Driving Licence FE. Is there an example of it in use?

Comment (merlinc) [reply lib/globals.js:120]: @MMcC-GDS Are you happy with this solution to your requirements?

Comment (MMcC-GDS) [reply lib/globals.js:120]: @merlinc yep, I'm happy for it to be merged if there are no issues with anyone else

See https://github.com/alphagov/di-ipv-cri-dl-front/pull/78/files

**Review by merlinc — APPROVED** [review]
Looks good on our side!

**Pull request merged**